practices/graded-browser-support.md

```diff
@@ -1,52 +1,48 @@
 # Graded Browser Support
 
-
 ## Our criteria for browser support
 
-
 We follow the guidelines for graded browser support outlined by Nate Koechley at Yahoo:
 [Yahoo Grade Browser Support guide](https://github.com/yui/yui3/wiki/Graded-Browser-Support)
 
-
-
 | Tables        | Grade-A support | Grade-C support  |
 | ------------- |:---------------:| ----------------:|
 | Chrome        | latest stable   |                  |
 | Edge          | latest stable   |                  |
-| Firefox       | latest stable   |               $1 |
+| Firefox       | latest stable   |               <2 |
```

Comment:
Is Firefox 2 still showing up in GA?  Are we applying the 1% rule, that we appear to be using, to Firefox as well?